config_serviceflags.go

```diff
@@ -35,6 +35,7 @@ type ServiceFlags struct {
 	CloudCleanupDisabled            bool `bson:"cloud_cleanup_disabled" json:"cloud_cleanup_disabled"`
 	ContainerConfigurationsDisabled bool `bson:"container_configurations_disabled" json:"container_configurations_disabled"`
 	SlackAppDisabled                bool `bson:"slack_app_disabled" json:"slack_app_disabled"`
+	RequireAuthAllRoutesDisabled    bool `bson:"require_auth_all_routes_disabled" json:"require_auth_all_routes_disabled"`
```

Comment:
Can we flip the wording of this route so that the false state is the default that we want? (since we do want this to be disabled to begin with) Like lightAuthDisabled or something. That or we can insert true into prod before we deploy so it'll start in the disabled state.